webroot/js/participantSummary.js

```diff
@@ -10,6 +20,37 @@ function loadAllSitesCallback(result) {
     siteList = JSON.parse(result).map((item) => { return { name: item.name, id: item.id, clientTypes: item.clientTypes } });
 };
 
+function setSearchValue(searchText) {
+    document.querySelector('#site-search').value = searchText;
+    $(resultsElement).hide();
+    $('#doSearch').click();
+}
+
+function searchSitesAutocomplete(e) {
+    
+    searchString = searchInput.value;
+    const options = {
+        threshold: .2,
```

Comment:
Curious - I see the default [threshold ](https://www.fusejs.io/api/options.html#threshold)is 0.6. Is there a good reason not to use that?

Reply:
I intentionally set it lower because I didn't see the point in getting auto-complete suggestions that are really not close to what we're looking for.  like being multiple letters off.  but we can change this if we think it's too restrictive.